### PR TITLE
#361: further update automation

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [x] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)

This PR covers part of issue #361 .

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -39,10 +31,12 @@ Files:
...
 
-1. Add a `generate_document_[LANG].sh` to the Tools folder
-2. Update `gitbookandepubandpdf.sh` with the language referenced
-3. Extend `Apply_Link_Check.sh` and `Apply_Linter_Check.sh`
-4. Update `gendocs.sh` and `gendoclsLocal.sh` so you can test it locally and see if the markdown land links make sense for the new language. Use this to file issues or report to the translator if fixes are necessary. Please check for pagination as well when you run it so that at release time, you have a proper PDF immediately
-5. Extend `../.travis.yml` to include the new language
-6. Update `../LANGS.md` to include the new language
-7. Extend the `../README.md` with the newly available language
+1. Create a folder with the language of choice.
+2. Extend `Apply_Link_Check.sh` and `Apply_Linter_Check.sh` with the new folder and make sure you do not end up with dead links or MD errors.
```
Comment (TheDauntless) [inline tools/README.md]:
```suggestion
2. Extend `Apply_Link_Check.sh` and `Apply_Linter_Check.sh` with the new folder and make sure you do not end up with dead links or Markdown errors.
```

File: tools/README.md [hunk 2]
```diff
@@ -39,10 +31,12 @@ Files:
...
+3. Update `.github/workflows/checkLint.yml` and add the new folder to the lint checker.
+4. Add the `LANGUAGE-METADATA` to the folder.
+5. Test the generation of the documentation by running `tools/docker/run_docker_masvs_generation_on_local.sh` and update `tools/docker/pandoc_makedocs.sh` where-ever necessary. See [the docker README.MD for more details on any necessary update process](docker/README.md).
```
Comment (TheDauntless) [inline tools/README.md]:
```suggestion
5. Test the generation of the document by running `tools/docker/run_docker_masvs_generation_on_local.sh` and update `tools/docker/pandoc_makedocs.sh` wherever necessary. See [the docker README.MD for more details on any necessary update processes](docker/README.md).
```

File: tools/README.md [hunk 3]
```diff
@@ -39,10 +31,12 @@ Files:
...
+6. Extend the folder in `export.py`
```
Comment (TheDauntless) [inline tools/README.md]: This seems a bit vague?

Comment (commjoen) [reply tools/README.md]: Fixed it!

**Pull request merged**